client/webserver/site/src/js/wallets.js

```diff
@@ -312,7 +312,7 @@ export default class WalletsPage extends BasePage {
   /* handleBalance handles notifications updating a wallet's balance. */
   handleBalanceNote (note) {
     const td = this.page.walletTable.querySelector(`[data-balance-target="${note.assetID}"]`)
-    td.textContent = (note.balances.zeroConf.available / 1e8).toFixed(8)
+    td.textContent = (note.balance.available / 1e8).toFixed(8)
```

Comment:
I'd be shocked if these are the only frontend changes.

Reply:
Amazingly, I went through registration and a trade, and saw no UI issues or errors in the browser debug console.